.travis.yml

```diff
@@ -80,7 +80,12 @@ before_install:
   - sudo apt-get install -y openmpi-bin libopenmpi-dev
 
   # Install nose framework for the Python testsuite.
+  # (Please note that the nose-capturestderr plugin is needed to suppress NEST output to stderr which may interfere 
+  #  with the nosetests output hindering correct log-parsing. stdout it captured by default.)
   - sudo apt-get install -y python-nose
+  - git clone https://github.com/sio2project/nose-capturestderr.git capturestderr
```

Comment:
@gtrensch : Have you tried to use the pypi  package for _nose-capturestderr 1.2_
https://pypi.python.org/pypi/nose-capturestderr/1.2